recipes/wt/all/conanfile.py

```diff
@@ -90,106 +125,172 @@ def _configure_cmake(self):
-            self._cmake.definitions['POSTGRES_FOUND'] = True
-        if self.settings.os == 'Windows':
-            self._cmake.definitions['CONNECTOR_FCGI'] = False
-            self._cmake.definitions['CONNECTOR_ISAPI'] = self.options.connector_isapi
+            self._cmake.definitions["OPENSSL_PREFIX"] = self.deps_cpp_info["openssl"].rootpath
+            self._cmake.definitions["OPENSSL_LIBRARIES"] = ";".join(_gather_libs("openssl"))
+            self._cmake.definitions["OPENSSL_INCLUDE_DIR"] = ";".join(self.deps_cpp_info["openssl"].include_paths)
+            self._cmake.definitions["OPENSSL_FOUND"] = True
+        if self.options.get_safe("with_mysql"):
+            self._cmake.definitions["MYSQL_LIBRARIES"] = ";".join(_gather_libs("libmysqlclient"))
+            self._cmake.definitions["MYSQL_INCLUDE"] = ";".join(self.deps_cpp_info["libmysqlclient"].include_paths)
+            self._cmake.definitions["MYSQL_DEFINITIONS"] = ";".join("-D%s" % d for d in self.deps_cpp_info["libmysqlclient"].defines)
+            self._cmake.definitions["MYSQL_FOUND"] = True
+        if self.options.get_safe("with_postgres"):
+            self._cmake.definitions["POSTGRES_LIBRARIES"] = ";".join(_gather_libs("libpq"))
+            self._cmake.definitions["POSTGRES_INCLUDE"] = ";".join(self.deps_cpp_info["libpq"].include_paths)
+            self._cmake.definitions["POSTGRES_FOUND"] = True
+        if self.options.get_safe("with_mssql") and self.settings.os != "Windows":
+            self._cmake.definitions["ODBC_LIBRARIES"] = ";".join(_gather_libs("odbc"))
+            self._cmake.definitions["ODBC_INCLUDE"] = ";".join(self.deps_cpp_info["odbc"].include_paths)
+            self._cmake.definitions["ODBC_FOUND"] = True
+        if self.settings.os == "Windows":
+            self._cmake.definitions["CONNECTOR_FCGI"] = False
+            self._cmake.definitions["CONNECTOR_ISAPI"] = self.options.connector_isapi
         else:
-            self._cmake.definitions['CONNECTOR_FCGI'] = self.options.connector_fcgi
-            self._cmake.definitions['CONNECTOR_ISAPI'] = False
+            self._cmake.definitions["CONNECTOR_FCGI"] = self.options.connector_fcgi
+            self._cmake.definitions["CONNECTOR_ISAPI"] = False
         self._cmake.configure(build_folder=self._build_subfolder)
         return self._cmake
 
     def build(self):
-        tools.replace_in_file(os.path.join(self._source_subfolder, 'CMakeLists.txt'), 'find_package(OpenSSL)', '#find_package(OpenSSL)')
-        tools.replace_in_file(os.path.join(self._source_subfolder, 'CMakeLists.txt'), 'INCLUDE(cmake/WtFindMysql.txt)', '#INCLUDE(cmake/WtFindMysql.txt)')
-        tools.replace_in_file(os.path.join(self._source_subfolder, 'CMakeLists.txt'), 'INCLUDE(cmake/WtFindPostgresql.txt)', '#INCLUDE(cmake/WtFindPostgresql.txt)')
+        self._validate_dependency_graph()
+        tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"), "find_package(OpenSSL)", "#find_package(OpenSSL)")
```

Comment:
Could the replace in file be moved to a patch?

Reply:
Not if it's a persistent behaviour in Wt's build files. Anyway there is already a lot of modifications in this PR ;)